### DISCUSSION TITLE
Waline 的手册好像有个写错的地方 || Waline’s manual seems to have a typo

### DISCUSSION THREAD

**Discussion by zimuwood** [walinejs/waline#2862] [General]
在 Waline 手册 Waline 客户端 V2 迁移指南中有关 placeholder 的部分写的是：placeholder: 使用 locales.placeholder，但在我实际配置的时候应该配置的是: locale.placeholder，这个地方是弄错了吗
![Quicker_20241216_000955](https://github.com/user-attachments/assets/e2a5afdd-b324-42da-aea0-890770a9a9b9)

---
In the Waline Manual Waline Client V2 Migration Guide, the part about placeholder says: placeholder: Use locales.placeholder, but when I actually configured it, I should configure it: locale.placeholder. Is this a mistake?
![Quicker_20241216_000955](https://github.com/user-attachments/assets/e2a5afdd-b324-42da-aea0-890770a9a9b9)

**Comment by lizheming** [comment]
已处理，感谢反馈

---
Processed, thanks for the feedback
